setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+from setuptools import setup
+import os
+
+
+def read(fname):
+    """
+    Read file (README).
+    """
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
```

Comment:
Doesn't that leave a dangling open file descriptor in python 2? Thought you needed to use a context manager to have this work as intended.